src/main/webapp/WEB-INF/themes/desktop/ftl/common/head.ftl

```diff
@@ -76,6 +76,8 @@
 <#include "baseJs.ftl" />
 <#include "additionalHeadTracking.ftl" />
 
+<script type="text/javascript" src="http://www.google.com/recaptcha/api/js/recaptcha_ajax.js"></script>
```

Comment:
I don't think we should have this in the head tag for all pages and definitely not in wombat.  Since we don't use captcha in all the pages.